src/nix/why-depends.cc

```diff
@@ -56,6 +57,12 @@ struct CmdWhyDepends : SourceExprCommand
             .description = "Show all edges in the dependency graph leading from *package* to *dependency*, rather than just a shortest path.",
             .handler = {&all, true},
         });
+
+        addFlag({
+            .longName = "precise",
+            .description = "For each edge of the graph, inspect the parent node to display the exact location in the path that causes the dependency",
```

Comment:
```suggestion
            .description = "For each edge of the graph, show the files in the parent that cause the dependency.",
```

Reply:
Thanks, I found the old description quite crappy but didn’t know how to put it right :)